DKV2/dkdbviews.cpp

```diff
@@ -17,11 +17,12 @@ SELECT
 
   ,IFNULL(Eingangsbuchung, '(steht aus)') AS Geldeingang
   ,IIF(V.zActive, ' aktiv ', ' ausgesetzt ') AS VerzinsungsStatus
-  ,IIF(V.Betrag <= IIF(V.thesaurierend == 0, IFNULL(summeAllerBuchungen, 0)
+  ,IIF(V.thesaurierend > 3, 'Fehler'
```

Comment:
there should be a "or v.thesaurierend < 0" case, too, right?

Reply:
Yes - that true!